pyveda/vedaset/veda/api.py

```diff
@@ -402,17 +402,26 @@ def from_id(cls, _id):
         doc['properties']['id'] = _id
         return cls.from_doc(doc)
 
+    # def __iter__(self):
```

Comment:
Can this commented out stuff be removed?

Reply:
yes